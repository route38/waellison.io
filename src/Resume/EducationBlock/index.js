import React from "react";
import School from "./School";

class EducationBlock extends React.Component {
    render() {
        let schools = this.props.schools;
        console.log(schools);
        return (
            <div>
                <h1>Education</h1>
                {
                    Object.keys(schools).map(key => (<School key={key} {...schools[key]} />))
                }
            </div>
        )
    }
}

export default EducationBlock;

