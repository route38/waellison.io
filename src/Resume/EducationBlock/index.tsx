import React from "react";
import School from "./School";
import { EducationSection } from "../../resumeData";

interface IProps {
    schools: EducationSection
}

class EducationBlock extends React.Component<IProps> {
  render() {
    let schools = this.props.schools;
    return (
      <div>
        <h2>Education</h2>
        {Object.keys(schools).map((key: string) => (
          <School key={key} {...schools[key]} />
        ))}
      </div>
    );
  }
}

export default EducationBlock;
