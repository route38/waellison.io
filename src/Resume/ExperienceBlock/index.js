import React from "react";
import Job from "./Job";

class ExperienceBlock extends React.Component {
    render() {
        let jobs = this.props.jobs;
        console.log(jobs);
        return (
            <div>
                <h2>Experience</h2>
                {
                    Object.keys(jobs).map(key => (<Job key={key} {...jobs[key]} />))
                }
            </div>
        )
    }
}

export default ExperienceBlock;
