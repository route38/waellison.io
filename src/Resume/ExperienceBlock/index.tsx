import React from "react";
import { resumeData, ExperienceSection } from "../../resumeData";
import Job from "./Job";

interface IProps {
    jobs: ExperienceSection;
}

class ExperienceBlock extends React.Component<IProps> {
  render() {
    let jobs = this.props.jobs;
    return (
      <div>
        <h2>Experience</h2>
        {Object.keys(jobs).map((key) => (
          <Job key={key} {...jobs[key]} />
        ))}
      </div>
    );
  }
}

export default ExperienceBlock;
