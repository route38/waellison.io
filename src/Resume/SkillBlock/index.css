#skillbox {
    display: flex;
    flex-flow: row wrap;
    justify-content: flex-start;
    align-items: stretch;
    align-content: stretch;
    margin: 0 -1rem;
}

ul.skills-learned-box {
    list-style-type: none;
    padding-left: 0;
    margin-left: 0;
}

ul.skills-learned-box li {
    display: inline-block;
    font-size: 9pt;
    background-color: #bbb;
    border-radius: 6px;
    height: 2rem;
    padding: 0 8px;
    font-family: "Montserrat", sans-serif;
    line-height: 2rem;
    margin-bottom: 4px;
}
