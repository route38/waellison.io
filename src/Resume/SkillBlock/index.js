import React from "react";
import SkillCategory from "./SkillCategory";

export class SkillBlock extends React.Component {
    render() {
        let skills = this.props.skills;
        return (
            <div id="skills">
                <h2>Skills</h2>
                <div id="skillbox">
                    {Object.keys(skills).map(category => {
                        return <SkillCategory
                            key={category}
                            children={skills[category]}
                            name={category} />;
                    })}
                </div>
            </div>
        );
    }
}

export default SkillBlock;
