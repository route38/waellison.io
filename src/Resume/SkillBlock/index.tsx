import React from "react";
import SkillCategory from "./SkillCategory";
import { SkillsSection } from "../../resumeData";

interface IProps {
    skills: SkillsSection;
}

export default class SkillBlock extends React.Component<IProps> {
  render() {
    let skills = this.props.skills;
    return (
      <div id="skills">
        <h2>Skills</h2>
        <div id="skillbox">
          {Object.keys(skills).map((category) => {
            return (
              <SkillCategory
                key={category}
                children={skills[category]}
                name={category}
              />
            );
          })}
        </div>
      </div>
    );
  }
}
