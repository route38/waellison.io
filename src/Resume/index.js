import React from 'react';
import resumeData from '../resumeData';
import Summary from './Summary';
import SkillBlock from './SkillBlock';
import EducationBlock from './EducationBlock';
import ExperienceBlock from './ExperienceBlock';

class Resume extends React.Component {
  render() {
    let resume = resumeData;
    return (
      <div className="container" id="resume">
        <Summary text={resume.summary} />
        <SkillBlock skills={resume.skills} />
        <ExperienceBlock jobs={resume.experience} />
        <EducationBlock schools={resume.education} />
      </div>
    );
  }
}

export default Resume;
