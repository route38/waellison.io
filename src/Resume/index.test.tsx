import React from "react";
import renderer from "react-test-renderer";
import Resume from "./index";

it("renders correct resume component", () => {
  const resumeRender: renderer.ReactTestRenderer | null = renderer.create(<Resume />);
  const tree: renderer.ReactTestRendererJSON | renderer.ReactTestRendererJSON[] | null = resumeRender.toJSON();
  expect(tree).toMatchSnapshot();
});
