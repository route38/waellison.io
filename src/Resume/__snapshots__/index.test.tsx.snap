// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correct resume component 1`] = `
<div
  className="container"
>
  <div
    id="summary"
  >
    <h1>
      William A. Ellison
    </h1>
    <h2>
      Summary
    </h2>
    <p
      dangerouslySetInnerHTML={
        Object {
          "__html": "I am a passionate, skilled software developer with a strong background in web development and three years' experience in healthcare revenue cycle management.  My skills include programming in Python and JavaScript with the Flask and React frameworks, PostgreSQL databases, and REST APIs.  I am also skilled at developing for the cloud with Docker, Kubernetes, and Microsoft Azure.  I have recognized skills in analysis, problem solving, training and coaching, and written and oral communication.",
        }
      }
    />
  </div>
  <div
    id="skills"
  >
    <h2>
      Skills
    </h2>
    <div
      id="skillbox"
    >
      <div
        className="skillCategory"
        id="DevOps"
      >
        <h3>
          DevOps
        </h3>
        <ul>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="Azure"
                className="captionedIcon"
                src="/icons/cloud-azure.svg"
              />
               
              Azure
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="Docker"
                className="captionedIcon"
                src="/icons/devops-docker.svg"
              />
               
              Docker
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="Git"
                className="captionedIcon"
                src="/icons/software-Git.svg"
              />
               
              Git
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="Linux"
                className="captionedIcon"
                src="/icons/skill-linux.svg"
              />
               
              Linux
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="Continuous integration"
                className="captionedIcon"
                src="/icons/skill-devops.svg"
              />
               
              <abbr
                title="Continuous integration"
              >
                CI
              </abbr>
              /
              <abbr
                title="Continuous delivery"
              >
                CD
              </abbr>
            </span>
          </li>
        </ul>
      </div>
      <div
        className="skillCategory"
        id="Libraries"
      >
        <h3>
          Libraries
        </h3>
        <ul>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="Flask"
                className="captionedIcon"
                src="/icons/framework-flask.svg"
              />
               
              Flask
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="jQuery"
                className="captionedIcon"
                src="/icons/library-jquery.svg"
              />
               
              jQuery
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="React"
                className="captionedIcon"
                src="/icons/library-react.svg"
              />
               
              React
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="RestApi"
                className="captionedIcon"
                src="/icons/skill-RestApi.svg"
              />
               
              REST APIs
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="PostgreSQL"
                className="captionedIcon"
                src="/icons/language-postgreSQL.svg"
              />
               
              PostgreSQL
            </span>
          </li>
        </ul>
      </div>
      <div
        className="skillCategory"
        id="Languages"
      >
        <h3>
          Languages
        </h3>
        <ul>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="Python"
                className="captionedIcon"
                src="/icons/language-python.svg"
              />
               
              Python
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="JavaScript"
                className="captionedIcon"
                src="/icons/language-javaScript.svg"
              />
               
              JavaScript
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="C++"
                className="captionedIcon"
                src="/icons/language-CeePlusPlus.svg"
              />
               
              C++
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="USA"
                className="captionedIcon"
                src="/icons/flag-USA.svg"
              />
               
              English
            </span>
          </li>
          <li
            className="captionedIconContainer"
          >
            <span>
              <img
                alt="French"
                className="captionedIcon"
                src="/icons/flag-france.svg"
              />
               
              French
            </span>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div>
    <h2>
      Education
    </h2>
    <div
      className="event"
    >
      <h3>
        freeCodeCamp
      </h3>
      <p>
        2022
        , 
        Certifications in Responsive Web Design and JavaScript Algorithms and Data Structures
      </p>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "Self-paced boot camp-style curriculum in major libraries used for frontend Web development, including jQuery, React, Redux, and Bootstrap.",
          }
        }
      />
      <ul
        className="skillsUsed"
      >
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "JavaScript",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "React.js",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Redux",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "HTML",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "CSS",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Bootstrap",
            }
          }
        />
      </ul>
    </div>
    <div
      className="event"
    >
      <h3>
        AlgoExpert
      </h3>
      <p>
        2022
        , 
        Certificate of Completion
      </p>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "Self-paced curriculum in algorithms and data structures, including linked lists, arrays, graphs, trees, strings, recursion, and dynamic programming.",
          }
        }
      />
      <ul
        className="skillsUsed"
      >
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Python",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "TypeScript",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Algorithms",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Data structures",
            }
          }
        />
      </ul>
    </div>
    <div
      className="event"
    >
      <h3>
        Nucamp
      </h3>
      <p>
        2021
        , 
        Certification in Backend Development with Python, SQL, and DevOps
      </p>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p>Boot camp course in modern backend Web development with the Python language, PostgreSQL, Flask, Django, and DevOps deployments to Microsoft Azure, Google Cloud, and Amazon AWS.</p><p>Offered coaching and assistance to classmates throughout the course.</p>",
          }
        }
      />
      <ul
        className="skillsUsed"
      >
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Python",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Flask",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "PostgreSQL",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Microsoft Azure",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Git",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "CI/CD",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Visual Studio Code",
            }
          }
        />
      </ul>
    </div>
    <div
      className="event"
    >
      <h3>
        Tennessee College of Applied Technology
      </h3>
      <p>
        2018
        , 
        Diploma in Medical Office Information Technology
      </p>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p>Broad-ranging instruction in concepts and procedures of the medical billing office, including reimbursement principles, medical terms, medical coding, and practice-management software.</p><p>Offered coaching and assistance to classmates throughout the course.</p>",
          }
        }
      />
      <ul
        className="skillsUsed"
      >
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Medical coding",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Medical billing",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Microsoft Word",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Microsoft Excel",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Medical terminology",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Teamwork",
            }
          }
        />
      </ul>
    </div>
    <div
      className="event"
    >
      <h3>
        University of Tennessee
      </h3>
      <p>
        2006 to 2011
        , 
        Bachelor of Science in Computer Science
      </p>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "Practical and theoretical instruction in computer science, including systems programming, object-oriented systems, graphical user interfaces, technical writing, and theoretical computer science, combined with traditional liberal-arts curriculum.",
          }
        }
      />
      <ul
        className="skillsUsed"
      >
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "C++",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Java",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Technical writing",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Teamwork",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Public speaking",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "LaTeX",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "OpenGL",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Subversion",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Emacs",
            }
          }
        />
      </ul>
    </div>
  </div>
  <div>
    <h2>
      Experience
    </h2>
    <div
      className="event"
    >
      <h3>
        Accounts Receivable Representative
      </h3>
      <p>
        2019
         to 
        present
         at 
        University Physicians' Association
      </p>
      <ul>
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Collaborated with a 10-member cross-functional team that reduced outstanding claims by 40% quarter-over-quarter, reducing A/R days by 33%",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Evaluated and resolved an average of 35 claims per day with 98% accuracy",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Assessed denied claims and reviewed for appeal processes",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Drafted appeal letters to all major carriers, overturning 70% of denials",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Recognized as my team's point of knowledge for questions on diagnosis and procedure coding",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Guided and trained personnel within a 55-member department, training on coding rules, standard workflows, software usage, and appeal letter origination",
            }
          }
        />
      </ul>
      <ul
        className="skillsUsed"
      >
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Telecommuting",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Medical coding",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Medical billing",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Writing",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Teamwork",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Time management",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Detail orientation",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Microsoft Excel",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Microsoft Word",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Microsoft Access",
            }
          }
        />
      </ul>
    </div>
    <div
      className="event"
    >
      <h3>
        Medical Billing Representative
      </h3>
      <p>
        2018
         to 
        2019
         at 
        University Physicians' Association
      </p>
      <ul>
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Partnered with an established 10-member team to review more than 8,000 outbound line items per day",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Reviewed 1,200 line items per day with 100% accuracy",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Increased clean claim rate by 5% on average by practice, reducing A/R overhead and turnaround time",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Analyzed an average of $100,000 of charges per day for inaccuracies",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Proposed workflow alterations which led to 10% faster turnaround of paper charge batches",
            }
          }
        />
      </ul>
      <ul
        className="skillsUsed"
      >
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Medical billing",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Medical coding",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Data entry",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Time management",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Teamwork",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Microsoft Excel",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Detail orientation",
            }
          }
        />
      </ul>
    </div>
    <div
      className="event"
    >
      <h3>
        Office Cleaner
      </h3>
      <p>
        2006
         to 
        2018
         at 
        We Clean It CJS
      </p>
      <ul>
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Supervised a two-person cleaning crew maintaining 50,000 square feet per night",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Led review initiatives of cleaning and training protocols, increasing productivity by 33%",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Redeveloped internal tracking systems to reduce lead time on bids by 50%",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Implemented a company website that doubled inbound sales calls",
            }
          }
        />
      </ul>
      <ul
        className="skillsUsed"
      >
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Time management",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Teamwork",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Detail orientation",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "WordPress",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "Linux",
            }
          }
        />
        <li
          className="skillTag"
          dangerouslySetInnerHTML={
            Object {
              "__html": "nginx",
            }
          }
        />
      </ul>
    </div>
  </div>
</div>
`;
